lib/util.js

```diff
@@ -19,8 +19,25 @@ var fs = require('fs'),
      */
     FILESIZE_OPTIONS = { spacer: '' },
 
-    PRO_API_HOST = 'api.getpostman.com',
-    USER_AGENT_VALUE = 'Newman/' + version;
+    POSTMAN_API_HOST = 'api.getpostman.com',
+
+    POSTMAN_API_URL = 'https://api.getpostman.com',
+
+    /**
+     * Map of resource type and its equivalent API pathname.
+     *
+     * @type {Object}
+     */
+    POSTMAN_API_PATH = {
+        collection: 'collections',
+        environment: 'environments'
+    },
+
+    API_KEY_HEADER = 'X-Api-Key',
+
+    USER_AGENT_VALUE = 'Newman/' + version,
+
+    UID_REGEX = /^[0-9A-Z]+-[0-9A-F]{8}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{4}-[0-9A-F]{12}$/i;
```

Comment:
Check if the API handles IDs in a case-insensitive manner.

Reply:
Yes, it has `ignoreCase` flag.